src/mocks/handlers.ts

```diff
@@ -364,7 +364,7 @@ export const handlers = [
       let results = structuredClone(checklistitems);
       if (filter) {
         // Filter for checklist items for a specific request
-        const RequestId = filter.match(/RequestId eq (.+)/);
+        const RequestId = filter.match(/RequestId eq (.+?)/);
```

Comment:
I like what you were trying to do here... should maybe be changed to \d+ instead of .+ because we are matching a number.